packages/perseus/src/widget-container.tsx

```diff
@@ -30,7 +30,6 @@ type DefaultProps = {
 
 type State = {
     sizeClass: "small" | "medium" | "large" | "xlarge";
-    widgetProps: WidgetProps<any, PerseusWidgetOptions>;
```

Comment:
So it seems like it wasn't just Renderer that was keeping a stash of RenderProps, but WidgetContainer too. Instead with this PR we're not just passing through WidgetProps and WidgetContainer is now a little dumber (_"it's a good thing"_). I got Jeremy's blessing, but lmk if you see a problem.